lib/bunyan.js

```diff
@@ -1088,7 +1088,11 @@ function safeCycles() {
 /**
  * XXX
  */
-if (mv) {
+if (!mv) {
+    RotatingFileStream = null
```

Comment:
This might work better if the `RotatingFileStream` variable were defined near the top of the file as `var RotatingFileStream = null`. Then have the `if` statement can assign the function to the variable if `mv` is defined.